packages/devtools_app/lib/src/screens/vm_developer/vm_object_model.dart

```diff
@@ -73,17 +80,65 @@ class ClassObject extends VmObject {
   @override
   SourceLocation? get _sourceLocation => obj.location;
 
+  ValueListenable<bool> get fetchingReachableSize => _fetchingReachableSize;
+  final ValueNotifier<bool> _fetchingReachableSize = ValueNotifier(false);
+  InstanceRef? get reachableSize => _reachableSize;
+  InstanceRef? _reachableSize;
+
+  ValueListenable<bool> get fetchingRetainedSize => _fetchingRetainedSize;
+  final ValueNotifier<bool> _fetchingRetainedSize = ValueNotifier(false);
+  InstanceRef? get retainedSize => _retainedSize;
+  InstanceRef? _retainedSize;
+
+  ValueListenable<bool> get fetchingRetainingPath => _fetchingRetainingPath;
```

Comment:
I think we can get rid of `ValueListenable<bool> get fetching* => ...` if we just change the getters for the data we're fetching to `ValueListenable<T?> get propertyName`. That way we can use `null` to specify that we're loading / haven't loaded, specifically for the data that's displayed in the expandable tiles.

Reply:
Done!